hw/ip/otbn/dv/rig/rig/gens/untaken_branch.py

```diff
@@ -0,0 +1,133 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import random
+from typing import Optional
+
+from shared.operand import ImmOperandType, RegOperandType
+from shared.insn_yaml import InsnsFile
+
+from ..config import Config
+from ..model import Model
+from ..program import ProgInsn, Program
+
+from ..snippet import ProgSnippet
+from ..snippet_gen import GenCont, GenRet, SnippetGen
+
+
+class UntakenBranch(SnippetGen):
+    '''A snippet generator that generates program with aligned or
+
+    unalligned addresses for which branches are not taken.
+
+    '''
```

Comment:
Docstrings should be formatted like Git commit messages: a short 1-line description, followed by zero or more paragraphs separated by blank lines. Here, maybe it's enough to say "Generate untaken branches"? (Note that we generate snippets here, not whole programs).